python/lsst/afw/detection/_footprintContinued.py

```diff
@@ -0,0 +1,68 @@
+# This file is part of afw.
+#
+# Developed for the LSST Data Management System.
+# This product includes software developed by the LSST Project
+# (https://www.lsst.org).
+# See the COPYRIGHT file at the top-level directory of this distribution
+# for details of code ownership.
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+
+from typing import TYPE_CHECKING
+
+if TYPE_CHECKING:
+    from lsst.afw.image import Image
```

Comment:
Do you actually need the `if` here? Isn't `TYPE_CHECKING` always set at this point?

Reply:
See previous comment about circular import introduced by the type hints.